driverdataaggregation/main.py

```diff
@@ -386,12 +411,29 @@ def aggregation_scaling_python(self,
                                     to request that the consumer application utilizes
                                     all physical cores for each client thread.
             :type cpu_hyperthreads: int, optional
+            :param languages: which language to use for the tester "cpp" or "fortran"
+            :type languages: str
+            :param wall_time: allotted time for database launcher to run
+            :type wall_time: str
             """
             logger.info("Starting dataset aggregation scaling with python tests")
             logger.info(f"Running with database backend: {get_db_backend()}")
             logger.info(f"Running with launcher: {launcher}")
 
-            exp = create_folder(exp_name, launcher)
+            exp, result_path = create_folder(exp_name, launcher)
+            
+            write_run_config(result_path,
+                    colocated=0,
```

Comment:
Is this intended for future use?

Reply:
The write run config at the beginning is placed in the top level directory of the run folder -> its so that users may go in and look at what flags they set for that run instead of going through each of the session folders and adding up all permutations